### ISSUE TITLE
Unable to use /rftdim safe del and dimension won't unload unable to safedel

### ISSUE THREAD

**Issue by GlitchOG** [McJty/RFToolsDimensions#274]
Issue description:  command /rftdim safedel  doesn't work on single player even with the config set to allow regular players to use the command, only works if player has cheats enabled. also when having the cheats enabled and trying to use the command it would say " dimension in use" even tho there's nothing in there, so i have to go to the main menu and load the world again to use the command. 

Versions:
* Minecraft: 1.12.2
* Forge: 2844
* CompatLayer (only if on Minecraft 1.10 or 1.11):
* McJtyLib: 3.5.3
* RFTools: 7.71
* RFTools Dimensions: 5.71

**Comment by josephcsible** [comment]
Can you confirm that when you run that command, you're not in the dimension, and neither are any chunkloaders?

**Comment by GlitchOG** [comment]
Yes , not inside the dimension and nothing running, i'm searching for specific worlds so i don't spend too much time inside a dimension. usually go in i dont like it. go out and delete. well in this case i can't delete so i have to keep making different dimensions IDs

**Comment by xXDarkNovaXx** [comment]
I can only confirm this problem
You have to go to the main menu and then reload the map so you can delete the dimension because otherwise he will say that the world is in use 
I just noticed this because I'm search dimlet in randomly created dimensions
